src/getEntityTypescriptPostgres.test.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+import { expect, test, describe } from "vitest";
+import { build_column } from "./getEntityTypescriptPostgres.js";
+
+describe("#unit-test build_column", () => {
+	test("string not nullable wo default val", () => {
+		const info = build_column({
```

Comment:
In general I think that all variables or functions should be camelCase

Reply:
I like snake_case more and more :)
But ok, I will try to no put too much in snake

Reply:
Its just the js standard
You see to be more python guy 🫢